build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         mavenCentral()
 
         maven {
-            url "http://repo.spring.io/milestone/"
+            url "https://repo.spring.io/milestone/"
```

Comment:
Is this repo used?

Reply:
it should be, Idea tried to download something from it. I will check.